File: cli/internal/infra/ui/ui.go
package ui

import (
	"fmt"
	"strings"

	"github.com/fatih/color"
)

var (
	blueArrow  = color.New(color.FgBlue, color.Bold).Sprint("==>")
	greenCheck = color.New(color.FgGreen, color.Bold).Sprint("✓")
	yellowBang = color.New(color.FgYellow, color.Bold).Sprint("!")
	grayBar    = color.New(color.FgWhite).Sprint(strings.Repeat("─", 64))
)

func Log(s string) {
	fmt.Printf("%s %s\n", blueArrow, s)
}

func Ok(s string) {
	fmt.Printf("%s   %s\n", greenCheck, s)
}

func Warn(s string) {
	fmt.Printf("%s   %s\n", yellowBang, s)
}

func Success(s string) {
	color.New(color.FgGreen, color.Bold).Println(s)
}

func Bar() {
	fmt.Println(grayBar)
}

File: cli-ts-legacy/src/infra/ui.ts
import chalk from 'chalk';

export const ui = {
  log(s: string) {
    console.log(`${chalk.blue.bold('==>')} ${s}`);
  },
  ok(s: string) {
    console.log(`${chalk.green.bold('✓')}   ${s}`);
  },
  warn(s: string) {
    console.log(`${chalk.yellow.bold('!')}   ${s}`);
  },
  success(s: string) {
    console.log(chalk.green.bold(s));
  },
  bar() {
    console.log(chalk.gray('─'.repeat(64)));
  },
};
